File: Python/3162_find_the_number_of_good_pairs_i.py
class Solution:
    def numberOfPairs(self, nums1: list[int], nums2: list[int], k: int) -> int:
        ret = 0
        for i in range(len(nums1)):
            for j in range(len(nums2)):
                if nums1[i] % (nums2[j] * k) == 0:
                    ret += 1
        return ret

File: JavaScript/3162_find_the_number_of_good_pairs_i.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @param {number} k
 * @return {number}
 */
var numberOfPairs = function(nums1, nums2, k) {
    let goodPairsCount = 0;
    for (const num1 of nums1) {
        for (const num2 of nums2) {
            if (num1 % (num2 * k) === 0) {
                goodPairsCount += 1;
            }
        }
    }
    return goodPairsCount;
};

